src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveEvaluator.cs

```diff
@@ -438,7 +438,22 @@ public virtual bool CanExecuteCode(string text)
 
         public Task<ExecutionResult> ResetAsync(bool initialize = true)
         {
-            GetInteractiveWindow().AddInput(_interactiveCommands.CommandPrefix + "reset");
+            // This is an implementation of ResetAsync on IInteractiveEvaluator interface.
```

Comment:
Consider using explicit interface implementation to make the purpose clearer and calling more difficult.